docs/elements/guides/sign-in.mdx

```diff
@@ -0,0 +1,267 @@
+---
+description: Learn how to build a complete sign-in form with Clerk Elements.
+---
+
+# Build a sign-in flow
+
+In this guide you will learn how to build a complete sign-in flow using Clerk Elements.
+
+<Callout type="info">
+
+Clerk Elements currently only works with Next.js and [Clerk Core 2](https://clerk.com/changelog/2024-04-19). As it gets closer to a stable release, support for additional frameworks will be added. If your Next.js application is already using Clerk, make sure to [upgrade to Core 2](/docs/upgrade-guides/core-2/nextjs). If you are starting from scratch, follow the [Next.js quickstart](/docs/quickstarts/nextjs) before proceeding.
+
+</Callout>
+
+<Steps>
+
+### Add a sign-in route
+
+Create a new route in your Next.js application. The route needs to be an [optional catch-all route](https://nextjs.org/docs/pages/building-your-application/routing/dynamic-routes#optional-catch-all-segments) so the sign-in flow can handled nested paths. Add the following snippet to `/app/sign-in/[[...sign-in]]/page.tsx`:
```

Comment:
```suggestion
Create a new route in your Next.js application. The route needs to be an [optional catch-all route](https://nextjs.org/docs/pages/building-your-application/routing/dynamic-routes#optional-catch-all-segments) so the sign-in flow can handled nested paths, as shown in the following example:
```
We should avoid repeating the file name to avoid introducing discrepancies if we ever edit this line but forget to edit the code sample (or vice versa)